template.tpl

```diff
@@ -60,59 +60,125 @@ ___TEMPLATE_PARAMETERS___
   },
   {
     "type": "TEXT",
-    "name": "currency",
-    "displayName": "Currency",
+    "name": "productIds",
+    "displayName": "Product IDs",
     "simpleValueType": true,
-    "help": "Currency in which the Order Value was paid",
-    "valueHint": "Currency of order, e.g. SEK, EUR, USD, etc",
+    "help": "A comma-separated string or an array of product IDs that were part of the Order.\nE.g.  \u0027123,456,789\u0027 or [\u0027123\u0027, \u0027456\u0027, \u0027789\u0027]",
+    "valueHint": "IDs of products in order, e.g. 123,456,789",
     "valueValidators": [
       {
         "type": "NON_EMPTY",
-        "errorMessage": "You must provide a Currency"
+        "errorMessage": "You must provide Product IDs"
       }
     ]
   },
   {
     "type": "TEXT",
-    "name": "productIds",
-    "displayName": "Product IDs",
+    "name": "currency",
+    "displayName": "Currency",
     "simpleValueType": true,
-    "help": "Comma-separated list of product IDs that were part of the Order",
-    "valueHint": "IDs of products in order, e.g. 123,456,789",
+    "help": "Currency in which the Order Value was paid. \nList of supported currencies is available on \u003ca href\u003d\u0027https://fixer.io/symbols\u0027\u003ehttps://fixer.io/symbols\u003c/a\u003e",
```

Comment:
What are these `https://fixer.io/` parts here?

Reply:
It goes to the information tooltip for the currency input field.
![image](https://user-images.githubusercontent.com/75425222/137095463-05ab461f-c899-4fd2-b7c3-e26f57e71e1c.png)

We use fixer.io API for validating the currency. Anything outside of this list will not work with currency exchange when generating the conversion stats.